src/org/werelate/wiki/PersonPageIndexer.java

```diff
@@ -26,7 +26,7 @@
 {
    private static final String XPATH_PERSON_SURNAME = "person/name/@surname | person/alt_name/@surname | person/name/@title_suffix | person/alt_name/@title_suffix";
    private static final String XPATH_PERSON_GIVENNAME = "person/name/@given | person/alt_name/@given | person/name/@title_prefix | person/alt_name/@title_prefix";
-   private static final Pattern YEAR_PATTERN = Pattern.compile("\\b(1\\d{3})\\b");
+   private static final Pattern YEAR_PATTERN = Pattern.compile("\\b([12]*\\d{3})\\b");  // changed to include 3-digit years and 2xxx (Dec 2020 by Janet Bjorndahl)
```

Comment:
Can you change this to [12]?  instead of [12]* ? Then we won't try to index 5-digit numbers like 12345 as years.

Reply:
My bad. I changed it to [012]? because I realized that if you re-index the whole file at some point, there will still be years with leading 0's.